app/services/dashboard/audit_service.rb

```diff
@@ -5,6 +5,7 @@ module Dashboard
   # methods pertaining to audit functionality for dashboard
   module AuditService
     include MoabOnStorageService
+    include InstrumentationSupport
```

Comment:
add comment here and in other places `# for caller method for annotating slow queries` or some such?  It might help us know where "caller" is defined.  Or maybe others would be comfy searching on "caller" to figure out where it's defined?  Oy.

Reply:
Happy to ask my IDE to figure this stuff out too